content/Video_Display.md

```diff
@@ -590,6 +590,18 @@ Whether the Window is displayed can be toggled using
 (See [above](#ff4a-wy-window-y-position-r-w-ff4b-wx-window-x-position-7-r-w)
 for the definition of "Window visibility".)
 
+::: tip Window Internal Line Counter
+
+The window keeps an internal line counter that's functionally similar to `LY`, and increments alongside it. However, it only gets incremented when the window is visible, as described [here](#ff4a-wy-window-y-position-r-w-ff4b-wx-window-x-position-7-r-w). That is whenever:
+
+1. `LCDC.5` is set (window enabled)
+2. `LY >= WY`
```

Comment:
This condition is not completely correct, WY just had to have been equal to LY at the start of mode 2 at some point in the frame.

Perhaps it better to not repeat the window activation conditions here and instead just have the link to the WX/WY section?